src/basic/errno-util.h

```diff
@@ -28,6 +31,11 @@ static inline int negative_errno(void) {
         return -errno;
 }
 
+static inline char *strerror_safe(int error) {
+        /* 'safe' here does NOT means thread safety. */
```

Comment:
Wouldn't it make sense to just use `strerror_r()` so can cover this *issue* as well?

Reply:
How can we determine the length of the error string?

Reply:
Ah, I knew there had to be a catch there, nevermind then :-)